### PR TITLE
chore: Download the `Cocoa SDK` instead of building it

### PR DESCRIPTION
**Pull request by bitsandfoxes**
#skip-changelog

### CODE REVIEW

File: test/Scripts.Tests/package-release.zip.snapshot [hunk 1]
```diff
@@ -40,8 +40,427 @@ Plugins/iOS/SentryNativeBridge.m.meta
...
 Plugins/iOS/Sentry.xcframework~/ios-arm64_x86_64-simulator/
+Plugins/iOS/Sentry.xcframework~/macos-arm64_x86_64/
+Plugins/iOS/Sentry.xcframework~/tvos-arm64/
+Plugins/iOS/Sentry.xcframework~/tvos-arm64_x86_64-simulator/
+Plugins/iOS/Sentry.xcframework~/watchos-arm64_arm64_32_armv7k/
+Plugins/iOS/Sentry.xcframework~/watchos-arm64_i386_x86_64-simulator/
+Plugins/iOS/Sentry.xcframework~/xros-arm64/
+Plugins/iOS/Sentry.xcframework~/xros-arm64_x86_64-simulator/
 Plugins/iOS/Sentry.xcframework~/Info.plist
+Plugins/iOS/Sentry.xcframework~/tvos-arm64/dSYMs/
```
Comment (vaind) [inline test/Scripts.Tests/package-release.zip.snapshot:53]: does it make sense to ship tvos and watchos?

Comment (vaind) [reply test/Scripts.Tests/package-release.zip.snapshot:53]: and xros, whatever that is

**Review by vaind — APPROVED** [review]
very nice!

File: test/Scripts.Tests/package-release.zip.snapshot (hunk 1 above)
Comment (brustolin) [reply test/Scripts.Tests/package-release.zip.snapshot:53]: > xros

visionOS

**Review by brustolin — APPROVED** [review]

File: test/Scripts.Tests/package-release.zip.snapshot (hunk 1 above)
Comment (bitsandfoxes) [reply test/Scripts.Tests/package-release.zip.snapshot:53]: Unity has these as build targets. We don't specifically have support for those (yet) but it doesn't make sense to me to fix (trim) up the `.xcframework`.
Screenshot 2024-10-21 at 10 27 13

**Comment by bruno-garcia** [comment]
@vaind does the dependency updater work on this now too?

**Comment by vaind** [comment]
> @vaind does the dependency updater work on this now too?

yes, it uses the .properties file: https://github.com/getsentry/sentry-unity/blob/03d455664c3f39ec9e00598f043e3ff427470655/modules/sentry-cocoa.properties

**Pull request merged**